docs/topics/modules/planning_development_work.adoc

```diff
@@ -0,0 +1,91 @@
+. Select the *Show Tree* check box to see the work items in a tree view. The tree view allows you to quickly add work items according to the guided hierarchy using the quick-add and in-line add options.
+. Use the quick-add option to create a *Scenario* as follows:
+.. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
++
+image::quickadd_wi_type.png[Quick Add work item type]
++
+.. In the *Work Item Title* field, add `Enable an end to end, quick, developer work-flow, within a continuous integration and deployment model` as the title.
+.. Press kbd:[Enter] to create the scenario.
++
+image::gs_scenario_eg.png[Example Scenario]
++
+
+. Similarly, select *Fundamental* in the *Add Work Item* pane, to create a fundamental work item with the title `The application should be secure with appropriate licenses`.
++
+image::gs_fundamental_eg.png[Example Fundamental]
++
+
+. Create child *Experience* and *Value Proposition* for the *Scenario* using the in-line add option as follows:
+.. In the work item list, click the (+) icon adjacent to the scenario to see the in-line *Add Child Work Item* pane below it.
++
+image::gs_inline-add_eg.png[Inline Add]
++
+
+.. Select *Experience* from the drop-down list, add the title `User should be able to launch any application through a simple work-flow`, and Press kbd:[Enter].
+.. Similarly, select *Value Proposition* and add the title `Easy end to end developer experience`.
++
+image::gs_exp_vp_egs.png[Example Experiences and Value Propositions]
++
+NOTE: A Scenario can have multiple realizing experiences and supporting value-propositions.
++
+
+. Create a child *Feature* to actualize the *Experience* using the in-line add option with the title `User should be able to easily launch a sample vertx application using a smooth developer work-flow`.
++
+image::gs_feature_eg.png[Example Feature]
+
+
+. Create the following child *Task* work items to implement the *Feature* using the in-line add option:
++
+* <<creating_new_project-hello-world,`Launch a sample quickstart application`>>
+* <<approving_build_pipeline,`Test the application in Stage and approve to the Run environment`>>
+* <<creating_che_workspace-hello-world,`Modify the quickstart codebase`>>
+* <<reviewing_publishing_changes-hello-world,`Review and publish changes to your codebase`>>
+
++
+Click the above tasks to see instructions on executing these tasks.
+
++
+image::gs_tasks_egs.png[Example Tasks]
+
+
+. Similarly, create a child Experience, Feature, and Task for the *Fundamental* - `The application should be secure with appropriate licenses`. The task could be titled <<analytics_stack_reports,`Use analytics to identify and resolve security and license issues in the application`>>.
++
+image::gs_fundamental_task_eg.png[Example Fundamental Task]
++
+For each of these tasks click on the work item to preview and:
```

Comment:
the and: is frowned upon in CCS (difficulties translating language fragments when translating and such). Can we complete this sentence here and then have independent phrases/sentence in the options below?